packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -268,10 +282,24 @@ class HierarchicalTermSelector extends Component {
 		const noParentOption = `— ${ parentSelectLabel } —`;
 		const newTermSubmitLabel = newTermButtonLabel;
 		const inputId = `editor-post-taxonomies__hierarchical-terms-input-${ instanceId }`;
+		const filterInputId = `editor-post-taxonomies__hierarchical-terms-filter-${ instanceId }`;
 
 		/* eslint-disable jsx-a11y/no-onchange */
 		return [
-			...this.renderTerms( availableTermsTree ),
+			<input
```

Comment:
the search input needs a visible `<label>` element, properly associated with for / id attributes